packages/zeta-app-contracts/scripts/multi-chain-value/search-zbtc-tx.ts

```diff
@@ -0,0 +1,199 @@
+import { ethers, network } from "hardhat";
+
+interface TransactionAnalysisResult {
+  amount: string;
+  btcAddress: string;
+  hash: string;
+  isSegWit: boolean;
+}
+
+// Specify the address you're interested in
+const ADDRESS = "0x13A0c5930C028511Dc02665E7285134B6d11A5f4"; // BTC.BTC
+const API_ENDPOINT = "zetachain.blockscout.com";
+const END_BLOCK = 1857360;
+
+const preCalculatedResults = [
```

Comment:
Nit: can we move this cache into another file so it doesn't affect the readability of the script?

Reply:
solved